src/core/lib/iomgr/port.h

```diff
@@ -131,23 +131,23 @@
 #define GRPC_POSIX_SOCKET 1
 #define GRPC_POSIX_SOCKETUTILS 1
 #define GRPC_POSIX_WAKEUP_FD 1
-#elif defined(GPR_OPENBSD)
+#elif defined(GPR_OPENBSD) /* defined(GRPC_CUSTOM_SOCKET) */
 #define GRPC_HAVE_IFADDRS 1
 #define GRPC_HAVE_IPV6_RECVPKTINFO 1
 #define GRPC_HAVE_UNIX_SOCKET 1
 #define GRPC_POSIX_NO_SPECIAL_WAKEUP_FD 1
 #define GRPC_POSIX_SOCKET 1
 #define GRPC_POSIX_SOCKETUTILS 1
 #define GRPC_POSIX_WAKEUP_FD 1
-#elif defined(GPR_NACL)
```

Comment:
I've never seen this type of thing before (the note after the ```#elif```).  I suspect it might add more confusion, unless its done at other points in our codebase.

Reply:
Yeah, I agree. I guess we need to come up with a standard. I wanted that would help easily match which elif endif matched. We already do it for ifdef and endif. I figured elif might be able to make use of the same.

Do you have an alternate suggestion?

Reply:
@vjpai Any ideas?

Reply:
I agree with Ken, it's kind of annoying. I would suggest that comments be in their own line, not part of a pre-processor line. Additionally, let the comment be a full sentence, like 

```c
/* This case implies that GRPC_CUSTOM_SOCKET is also defined */
```

or whatever as appropriate.